src/components/SectionHeader/SectionHeader.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { HTMLAttributes, PropsWithChildren } from "react";
+import styles from "./SectionHeader.module.css";
+
+const SectionHeader: React.FC<HTMLAttributes<PropsWithChildren>> = ({
+  children,
+  className,
+}) => {
+  return (
+    <div className={`${styles.header} ion-margin ${className}`}>{children}</div>
```

Comment:
Oh! Is this how scoping of styles works in this framework? I had been [prefixing the class names](https://github.com/microbiomedata/nmdc-field-notes/blob/main/src/pages/WelcomePage/WelcomePage.css) in an attempt to avoid styles designed for one component impacting other components. I didn't know the framework came with a ready-to-use scoping solution.

Reply:
It's not an Ionic-specific thing, so I'm not sure they even mention it in their docs. But Ionic's build tooling is based on around Vite, and [Vite supports CSS Modules](https://vitejs.dev/guide/features.html#css-modules) out of the box.

Reply:
Thanks! I had both seen that syntax before (on another project) and come across that name ("CSS Modules") before, but didn't know they were the same thing, and also didn't know that was something we could use out of the box. I'll create an issue for myself to update the "legacy" CSS I wrote a few days ago, to use CSS Modules. Resolving this thread.